drivers/SmartThings/matter-sensor/profiles/aqs-temp-humidity-co2-pm25-tvoc-meas.yml

```diff
@@ -0,0 +1,22 @@
+name: aqs-temp-humidity-co2-pm25-tvoc-meas
+components:
+  - id: main
+    capabilities:
+      - id: airQualityHealthConcern
+        version: 1
+      - id: temperatureMeasurement
+        version: 1
+      - id: relativeHumidityMeasurement
+        version: 1
+      - id: carbonDioxideMeasurement
+        version: 1
+      - id: dustSensor
```

Comment:
`dustSensor` has both the `dustLevel` and `fineDustLevel` attributes, but this profile is only for PM2.5 (fine dust level). We either need to use an embedded device config to hide `dustLevel` from the detail page and automation builder, or switch to the `fineDustSensor` capability which contains only the `fineDustLevel` attribute

Reply:
Resolved: [d2a8b17](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1422/commits/d2a8b17af05af62381f52c59024e675afcf5136b)